src/build/npm/jobs/npm-package.job.yaml

```diff
@@ -1,17 +1,63 @@
+# Job to build an NPM Package
 parameters:
-  path: ''
-  test: true
-  displayName: 'Build NPM'
-  skipDependencyChecks: false
+  - name: displayName
+    default: 'Build NPM'
+  - name: path
+    default: ''
+  - name: test
+    default: true
```

Comment:
IMO test should default to false if something is being used for CICD by default, rather than build validation - happy to discuss

Same comment for other packaging jobs

Reply:
As discussed, will leave this as true for now. This job would be used in both PR validation and CICD, further examples can be provided for separate pipelines.

I think it is better that tests run too often than not at all if this is defaulted to `false`.